.gitlab-ci.yml

```diff
@@ -79,6 +79,19 @@ make-coq-latest:
       - tags
       - merge_requests
 
+.dune-build:
+  stage: build
+  image: coqorg/coq:dev-ocaml-4.13.1-flambda
+  script:
+    - opam switch "$COMPILER_EDGE"
+    - opam install -y dune
+    - opam pin -y add dune git+https://github.com/Alizter/dune#coq-compose.git
```

Comment:
Is this branch of dune a requirement?

Reply:
To be clear, detection of libraries in user-contrib hasn't been implemented yet, only inter-project composition of coq dune projects. Adding deps on installed libs is the next step, but I am not certain if this will make 3.3.